orbbec_camera/component_descriptions/orbbec_camera_ob_camera_node.json

```
@@ -2,30 +2,60 @@
     "$schema": "https://docs.aica.tech/schemas/1-1-1/component.schema.json",
     "name": "Orbbec Camera",
     "description": {
-        "brief": "Stream color and depth images from an Orbbec camera"
+        "brief": "Stream color and depth images from an Orbbec camera."
```

```suggestion
        "brief": "Stream color and depth images from an Orbbec camera",
        "details": ...
```
Could you come up with a detailed description here please @SprGrf ? Examples can be found in the core collections